data/drumkits/TR808EmulationKit/drumkit.xml

```diff
@@ -2,12 +2,16 @@
 <drumkit_info xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.hydrogen-music.org/drumkit">
  <name>TR808EmulationKit</name>
  <author>ArtemioLabs [http://artemiolabs.com]</author>
- <info>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
-&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
-p, li { white-space: pre-wrap; }
-&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:10pt; font-weight:400; font-style:normal;">
-&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This drum kit contains 16 original sounds that were synthesized from scratch (using only basic synth waves, like sine, noise etc.). It simulates the sounds of the legendary Roland TR-808 Rhythm Composer. More kits from ArtemioLabs: http://artemiolabs.com/downloads/audio/hydrogen/&lt;/p>&lt;/body>&lt;/html></info>
+ <info>This drum kit contains 16 original sounds that were synthesized from scratch (using only basic synth waves, like sine, noise etc.). It simulates the sounds of the legendary Roland TR-808 Rhythm Composer. More kits from ArtemioLabs: http://artemiolabs.com/downloads/audio/hydrogen/</info>
  <license>GPL</license>
+ <image>Roland_TR-808_drum_machine.jpg</image>
```

Comment:
I'll verify to be sure - I forgot about this one, as I had updated it before adding the imagelicense tag.
Confirmed: it is at https://commons.wikimedia.org/wiki/File:Roland_TR-808_drum_machine.jpg and the license/usage info is:
`Eriq at Dutch Wikipedia [GFDL (http://www.gnu.org/copyleft/fdl.html) or CC-BY-SA-3.0 (http://creativecommons.org/licenses/by-sa/3.0/)], via Wikimedia Commons`
 I'll admit to not being 100% clear on how you can or cannot use GFDL or CC licensed stuff in GPL projects